_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,468 @@
+/* ============================
+-------------------------------
+ MIXINS
+ 1. Font Sizing
+ 2. Line Height
+ 3. Placeholder Styling
+ 4. Flex Support
+ 5. Headings
+
+ EXTENDS
+ 6. Headings
+ 7. Flex Polyfill
+ 8. Wrappers
+ 9. Windows
+ 10. Visual Disabled
+ 11. Layering/Zindexing
+ 12. Clear Children
+ 13. Buttons
+ 14. Wells
+ 15. Lists
+ 16. Icon Fonts
+ 17. Accessibility
+ 18. Text Handling
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Font Sizing
+---------------------------- */
+@mixin font-size($size: 16) {
+    font-size: ($size / 10) + rem;
+}
+
+
+/* ============================
+2. Line Height
+---------------------------- */
+@mixin line-height($size: 1.5rem) {
+    line-height: (($size / 10) * 1.48) + rem;
+}
+
+
+/* ============================
+3. Placeholder Styling
+---------------------------- */
+@mixin placeholder($color) {
+    :-moz-placeholder {
+        color: $color;
+    }
+
+    ::-webkit-input-placeholder {
+        color: $color;
+    }
+
+    :-ms-input-placeholder {
+         color: $color;
+    }
+}
+
+
+/* ============================
+4. Flex Support
+---------------------------- */
+@mixin ui-flexbox() {
+    display: -webkit-flex;
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: flex;
+}
+
+
+/* ============================
+5. Headings
+---------------------------- */
+@mixin page-title($type) {
+    @extend %heading;
+    @include font-size(30);
+    // font-family: $font-$type;
+}
+
+@mixin page-subtitle($type) {
+    @extend %heading;
+    @include font-size(22);
+    // font-family: $font-$type;
+}
+
+@mixin section-title($type) {
+    @extend %heading;
+    @include font-size(20);
+    // font-family: $font-$type;
+}
+
+@mixin section-subtitle($type) {
+    @extend %heading;
+    @include font-size(18);
+    // font-family: $font-$type;
+}
+
+@mixin heading-general($type) {
+    @extend %heading;
+    @include font-size(16);
+    // font-family: $font-$type;
+}
+
+
+/* ============================
+6. Headings
+---------------------------- */
+%heading {
+    font-weight: 600;
+}
+
+
+/* ============================
+7. Flex Polyfill
+---------------------------- */
+%justify-right-flex {
+    -webkit-box-pack: flex-end;
+    -webkit-justify-content: flex-end;
+    -moz-box-pack: flex-end;
+    -ms-flex-pack: flex-end;
+    justify-content: flex-end;
+}
+
+%justify-left-flex {
+    -webkit-justify-content: flex-start;
+    -webkit-box-pack: flex-start;
+    -moz-box-pack: flex-start;
+    -ms-flex-pack: flex-start;
+    justify-content: flex-start;
+}
+
+%align-center-flex {
+    -webkit-align-items: center;
+    -webkit-flex-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+
+/* ============================
+8. Wrappers
+---------------------------- */
+%wrapper {
+    @include clearfix();
+    @include box-sizing(border-box);
+    width: 100%;
+}
+
+.col-wide  {
```

Comment:
Please remove. This looks like an artifact from Studio's base styling and will be covered with layout/grid-based stories.